<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Mail</class>
 <widget class="QDialog" name="Mail">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>500</width>
    <height>700</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>500</width>
    <height>700</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>500</width>
    <height>700</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QFrame" name="frame">
     <property name="styleSheet">
      <string notr="true">background-color: rgb(45, 45, 45);</string>
     </property>
     <property name="frameShape">
      <enum>QFrame::NoFrame</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_4">
      <property name="spacing">
       <number>0</number>
      </property>
      <property name="leftMargin">
       <number>0</number>
      </property>
      <property name="topMargin">
       <number>0</number>
      </property>
      <property name="rightMargin">
       <number>0</number>
      </property>
      <property name="bottomMargin">
       <number>0</number>
      </property>
      <item>
       <widget class="QFrame" name="frame_2">
        <property name="styleSheet">
         <string notr="true">QFrame{
	background-color: rgb(35, 35, 35);
	border-radius:5px;
}</string>
        </property>
        <property name="frameShape">
         <enum>QFrame::NoFrame</enum>
        </property>
        <property name="frameShadow">
         <enum>QFrame::Raised</enum>
        </property>
        <layout class="QVBoxLayout" name="verticalLayout_2">
         <property name="leftMargin">
          <number>0</number>
         </property>
         <property name="topMargin">
          <number>0</number>
         </property>
         <property name="rightMargin">
          <number>0</number>
         </property>
         <property name="bottomMargin">
          <number>0</number>
         </property>
         <item>
          <widget class="QFrame" name="frame_3">
           <property name="minimumSize">
            <size>
             <width>0</width>
             <height>40</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>16777215</width>
             <height>30</height>
            </size>
           </property>
           <property name="frameShape">
            <enum>QFrame::NoFrame</enum>
           </property>
           <property name="frameShadow">
            <enum>QFrame::Raised</enum>
           </property>
           <layout class="QHBoxLayout" name="horizontalLayout">
            <property name="topMargin">
             <number>0</number>
            </property>
            <property name="bottomMargin">
             <number>0</number>
            </property>
            <item>
             <widget class="QLabel" name="label">
              <property name="minimumSize">
               <size>
                <width>60</width>
                <height>0</height>
               </size>
              </property>
              <property name="maximumSize">
               <size>
                <width>60</width>
                <height>16777215</height>
               </size>
              </property>
              <property name="text">
               <string/>
              </property>
              <property name="pixmap">
               <pixmap resource="../resource/asset.qrc">:/icons/asset/mail.svg</pixmap>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLabel" name="label_2">
              <property name="font">
               <font>
                <family>Arial</family>
                <pointsize>10</pointsize>
               </font>
              </property>
              <property name="styleSheet">
               <string notr="true">QLabel{
	color: rgb(255, 255, 255);
}</string>
              </property>
              <property name="text">
               <string>Email Application Data</string>
              </property>
              <property name="alignment">
               <set>Qt::AlignCenter</set>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QPushButton" name="btn_minimize">
              <property name="minimumSize">
               <size>
                <width>16</width>
                <height>16</height>
               </size>
              </property>
              <property name="maximumSize">
               <size>
                <width>17</width>
                <height>17</height>
               </size>
              </property>
              <property name="styleSheet">
               <string notr="true">QPushButton{
	background-color: rgb(85, 255, 127);
	border:none;
	border-radius: 8px;
}

QPushButton:hover{	
	background-color: rgba(85, 255, 127,150);
	
}</string>
              </property>
              <property name="text">
               <string/>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QPushButton" name="btn_maximize">
              <property name="minimumSize">
               <size>
                <width>16</width>
                <height>16</height>
               </size>
              </property>
              <property name="maximumSize">
               <size>
                <width>17</width>
                <height>17</height>
               </size>
              </property>
              <property name="styleSheet">
               <string notr="true">QPushButton{
	
	background-color: rgb(255, 170, 0);
	border:none;
	border-radius: 8px;
}

QPushButton:hover{	
	background-color: rgba(255, 170, 0,150);
	
}</string>
              </property>
              <property name="text">
               <string/>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QPushButton" name="btn_close">
              <property name="minimumSize">
               <size>
                <width>16</width>
                <height>16</height>
               </size>
              </property>
              <property name="maximumSize">
               <size>
                <width>17</width>
                <height>17</height>
               </size>
              </property>
              <property name="styleSheet">
               <string notr="true">QPushButton{
	background-color: rgb(255, 0, 0);
	border:none;
	border-radius: 8px;
}

QPushButton:hover{	
	background-color: rgba(255, 0, 0,150);
	
}</string>
              </property>
              <property name="text">
               <string/>
              </property>
             </widget>
            </item>
           </layout>
          </widget>
         </item>
         <item>
          <widget class="QFrame" name="frame_4">
           <property name="frameShape">
            <enum>QFrame::NoFrame</enum>
           </property>
           <property name="frameShadow">
            <enum>QFrame::Raised</enum>
           </property>
           <widget class="QLabel" name="label_36">
            <property name="geometry">
             <rect>
              <x>30</x>
              <y>120</y>
              <width>31</width>
              <height>31</height>
             </rect>
            </property>
            <property name="styleSheet">
             <string notr="true">background-color: rgb(45, 45, 45);</string>
            </property>
            <property name="text">
             <string/>
            </property>
            <property name="pixmap">
             <pixmap resource="../resource/asset.qrc">:/icons/asset/user.svg</pixmap>
            </property>
           </widget>
           <widget class="QLabel" name="label_37">
            <property name="geometry">
             <rect>
              <x>30</x>
              <y>330</y>
              <width>31</width>
              <height>31</height>
             </rect>
            </property>
            <property name="styleSheet">
             <string notr="true">background-color: rgb(45, 45, 45);</string>
            </property>
            <property name="text">
             <string/>
            </property>
            <property name="pixmap">
             <pixmap resource="../resource/asset.qrc">:/icons/asset/lock.svg</pixmap>
            </property>
           </widget>
           <widget class="QLabel" name="label_notification">
            <property name="geometry">
             <rect>
              <x>20</x>
              <y>530</y>
              <width>441</width>
              <height>81</height>
             </rect>
            </property>
            <property name="font">
             <font>
              <family>MS Shell Dlg 2</family>
              <pointsize>10</pointsize>
              <weight>50</weight>
              <bold>false</bold>
             </font>
            </property>
            <property name="styleSheet">
             <string notr="true">QLabel{
	color: rgb(255, 255, 255);
	background-color: rgb(45, 45, 45);
	border-radius: 10px;
}</string>
            </property>
            <property name="text">
             <string>Notification</string>
            </property>
            <property name="alignment">
             <set>Qt::AlignCenter</set>
            </property>
            <property name="wordWrap">
             <bool>true</bool>
            </property>
           </widget>
           <widget class="QLineEdit" name="sender_password">
            <property name="geometry">
             <rect>
              <x>17</x>
              <y>320</y>
              <width>441</width>
              <height>51</height>
             </rect>
            </property>
            <property name="minimumSize">
             <size>
              <width>0</width>
              <height>0</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>16777215</width>
              <height>16777215</height>
             </size>
            </property>
            <property name="font">
             <font>
              <pointsize>10</pointsize>
             </font>
            </property>
            <property name="styleSheet">
             <string notr="true">QLineEdit{
	color: rgb(255, 255, 255);
	border: 2px solid  rgb(45, 45, 45);
	border-radius:15px;
	padding-left: 50px;
}

QLineEdit:hover{
	border:2px solid rgb(35, 35, 35);
}

QLineEdit:focus{
	border:2px solid rgb(255, 255, 255);
}</string>
            </property>
            <property name="echoMode">
             <enum>QLineEdit::Password</enum>
            </property>
            <property name="placeholderText">
             <string>Password</string>
            </property>
            <property name="clearButtonEnabled">
             <bool>true</bool>
            </property>
           </widget>
           <widget class="QLineEdit" name="email_subject">
            <property name="geometry">
             <rect>
              <x>20</x>
              <y>180</y>
              <width>441</width>
              <height>51</height>
             </rect>
            </property>
            <property name="minimumSize">
             <size>
              <width>0</width>
              <height>0</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>16777215</width>
              <height>16777215</height>
             </size>
            </property>
            <property name="font">
             <font>
              <pointsize>10</pointsize>
             </font>
            </property>
            <property name="styleSheet">
             <string notr="true">QLineEdit{
	color: rgb(255, 255, 255);
	border: 2px solid  rgb(45, 45, 45);
	border-radius:15px;
	padding-left: 45px;
}

QLineEdit:hover{
	border:2px solid rgb(35, 35, 35);
}

QLineEdit:focus{
	border:2px solid rgb(255, 255, 255);
}</string>
            </property>
            <property name="placeholderText">
             <string>Subject</string>
            </property>
            <property name="clearButtonEnabled">
             <bool>true</bool>
            </property>
           </widget>
           <widget class="QLineEdit" name="email_sender">
            <property name="geometry">
             <rect>
              <x>17</x>
              <y>250</y>
              <width>441</width>
              <height>51</height>
             </rect>
            </property>
            <property name="minimumSize">
             <size>
              <width>0</width>
              <height>0</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>16777215</width>
              <height>16777215</height>
             </size>
            </property>
            <property name="font">
             <font>
              <pointsize>10</pointsize>
             </font>
            </property>
            <property name="styleSheet">
             <string notr="true">QLineEdit{
	color: rgb(255, 255, 255);
	border: 2px solid  rgb(45, 45, 45);
	border-radius:15px;
	padding-left: 50px;
}

QLineEdit:hover{
	border:2px solid rgb(35, 35, 35);
}

QLineEdit:focus{
	border:2px solid rgb(255, 255, 255);
}</string>
            </property>
            <property name="placeholderText">
             <string>Email</string>
            </property>
            <property name="clearButtonEnabled">
             <bool>true</bool>
            </property>
           </widget>
           <widget class="QLineEdit" name="email_from">
            <property name="geometry">
             <rect>
              <x>20</x>
              <y>110</y>
              <width>441</width>
              <height>51</height>
             </rect>
            </property>
            <property name="minimumSize">
             <size>
              <width>338</width>
              <height>0</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>16777215</width>
              <height>16777215</height>
             </size>
            </property>
            <property name="font">
             <font>
              <pointsize>10</pointsize>
             </font>
            </property>
            <property name="styleSheet">
             <string notr="true">QLineEdit{
	color: rgb(255, 255, 255);
	border: 2px solid  rgb(45, 45, 45);
	border-radius:15px;
	padding-left: 45px;
}

QLineEdit:hover{
	border:2px solid rgb(35, 35, 35);
}

QLineEdit:focus{
	border:2px solid rgb(255, 255, 255);
}</string>
            </property>
            <property name="placeholderText">
             <string>From</string>
            </property>
            <property name="clearButtonEnabled">
             <bool>true</bool>
            </property>
           </widget>
           <widget class="QLabel" name="label_38">
            <property name="geometry">
             <rect>
              <x>30</x>
              <y>190</y>
              <width>31</width>
              <height>31</height>
             </rect>
            </property>
            <property name="styleSheet">
             <string notr="true">background-color: rgb(45, 45, 45);</string>
            </property>
            <property name="text">
             <string/>
            </property>
            <property name="pixmap">
             <pixmap resource="../resource/asset.qrc">:/icons/asset/edit-3.svg</pixmap>
            </property>
           </widget>
           <widget class="QLabel" name="label_39">
            <property name="geometry">
             <rect>
              <x>30</x>
              <y>260</y>
              <width>31</width>
              <height>31</height>
             </rect>
            </property>
            <property name="styleSheet">
             <string notr="true">background-color: rgb(45, 45, 45);</string>
            </property>
            <property name="text">
             <string/>
            </property>
            <property name="pixmap">
             <pixmap resource="../resource/asset.qrc">:/icons/asset/mail.svg</pixmap>
            </property>
           </widget>
           <widget class="QLineEdit" name="reg_email">
            <property name="geometry">
             <rect>
              <x>20</x>
              <y>40</y>
              <width>441</width>
              <height>51</height>
             </rect>
            </property>
            <property name="font">
             <font>
              <pointsize>10</pointsize>
             </font>
            </property>
            <property name="styleSheet">
             <string notr="true">QLineEdit{
	color: rgb(255, 255, 255);
	border: 2px solid  rgb(45, 45, 45);
	border-radius:15px;
	padding-left: 45px;
}

QLineEdit:hover{
	border:2px solid rgb(35, 35, 35);
}

QLineEdit:focus{
	border:2px solid rgb(255, 255, 255);
}</string>
            </property>
            <property name="placeholderText">
             <string>Receipient(s) </string>
            </property>
            <property name="clearButtonEnabled">
             <bool>true</bool>
            </property>
           </widget>
           <widget class="QLabel" name="label_44">
            <property name="geometry">
             <rect>
              <x>30</x>
              <y>50</y>
              <width>31</width>
              <height>31</height>
             </rect>
            </property>
            <property name="text">
             <string/>
            </property>
            <property name="pixmap">
             <pixmap resource="../resource/asset.qrc">:/icons/asset/mail.svg</pixmap>
            </property>
           </widget>
           <widget class="QLineEdit" name="image_file_reg">
            <property name="geometry">
             <rect>
              <x>20</x>
              <y>390</y>
              <width>251</width>
              <height>51</height>
             </rect>
            </property>
            <property name="font">
             <font>
              <pointsize>10</pointsize>
             </font>
            </property>
            <property name="styleSheet">
             <string notr="true">QLineEdit{
	color: rgb(255, 255, 255);
	border: 2px solid  rgb(45, 45, 45);
	border-radius:10px;
	padding-left: 10px;
}

QLineEdit:hover{
	border:2px solid rgb(35, 35, 35);
}

QLineEdit:focus{
	border:2px solid rgb(255, 255, 255);
}</string>
            </property>
            <property name="placeholderText">
             <string>File path</string>
            </property>
            <property name="clearButtonEnabled">
             <bool>true</bool>
            </property>
           </widget>
           <widget class="QPushButton" name="btn_browse_reg">
            <property name="geometry">
             <rect>
              <x>20</x>
              <y>460</y>
              <width>211</width>
              <height>51</height>
             </rect>
            </property>
            <property name="font">
             <font>
              <pointsize>10</pointsize>
             </font>
            </property>
            <property name="styleSheet">
             <string notr="true">QPushButton{
	color: rgb(255, 255, 255);
	background-color: rgb(45, 45, 45);
	border:none;
	border-radius: 10px;
}

QPushButton:hover{
	border-radius: 20px;
}

QPushButton:pressed{
	border: 2px solid rgb(255,255,255);	
}</string>
            </property>
            <property name="text">
             <string>Browse</string>
            </property>
            <property name="icon">
             <iconset resource="../resource/asset.qrc">
              <normaloff>:/icons/asset/upload.svg</normaloff>:/icons/asset/upload.svg</iconset>
            </property>
            <property name="iconSize">
             <size>
              <width>30</width>
              <height>30</height>
             </size>
            </property>
            <property name="flat">
             <bool>true</bool>
            </property>
           </widget>
           <widget class="QPushButton" name="btn_send_mail">
            <property name="geometry">
             <rect>
              <x>250</x>
              <y>460</y>
              <width>211</width>
              <height>51</height>
             </rect>
            </property>
            <property name="font">
             <font>
              <pointsize>10</pointsize>
             </font>
            </property>
            <property name="styleSheet">
             <string notr="true">QPushButton{
	color: rgb(255, 255, 255);
	background-color: rgb(45, 45, 45);
	border:none;
	border-radius: 10px;
}

QPushButton:hover{
	border-radius: 20px;
}

QPushButton:pressed{
	border: 2px solid rgb(255,255,255);	
}</string>
            </property>
            <property name="text">
             <string>Send mail</string>
            </property>
            <property name="icon">
             <iconset resource="../resource/asset.qrc">
              <normaloff>:/icons/asset/send.svg</normaloff>:/icons/asset/send.svg</iconset>
            </property>
            <property name="iconSize">
             <size>
              <width>30</width>
              <height>30</height>
             </size>
            </property>
            <property name="flat">
             <bool>true</bool>
            </property>
           </widget>
           <widget class="QLineEdit" name="rep_name">
            <property name="geometry">
             <rect>
              <x>280</x>
              <y>390</y>
              <width>181</width>
              <height>51</height>
             </rect>
            </property>
            <property name="font">
             <font>
              <pointsize>10</pointsize>
             </font>
            </property>
            <property name="styleSheet">
             <string notr="true">QLineEdit{
	color: rgb(255, 255, 255);
	border: 2px solid  rgb(45, 45, 45);
	border-radius:10px;
	padding-left: 10px;
}

QLineEdit:hover{
	border:2px solid rgb(35, 35, 35);
}

QLineEdit:focus{
	border:2px solid rgb(255, 255, 255);
}</string>
            </property>
            <property name="placeholderText">
             <string>Receipient name</string>
            </property>
            <property name="clearButtonEnabled">
             <bool>true</bool>
            </property>
           </widget>
           <zorder>sender_password</zorder>
           <zorder>email_from</zorder>
           <zorder>label_36</zorder>
           <zorder>label_37</zorder>
           <zorder>label_notification</zorder>
           <zorder>email_subject</zorder>
           <zorder>email_sender</zorder>
           <zorder>label_38</zorder>
           <zorder>label_39</zorder>
           <zorder>reg_email</zorder>
           <zorder>label_44</zorder>
           <zorder>image_file_reg</zorder>
           <zorder>btn_browse_reg</zorder>
           <zorder>btn_send_mail</zorder>
           <zorder>rep_name</zorder>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="../resource/asset.qrc"/>
 </resources>
 <connections/>
</ui>
